lib/ReactViews/Workbench/WorkbenchItem.tsx

```diff
@@ -22,187 +21,167 @@ import { TextSpan } from "../../Styled/Text";
+              <BoxSpan paddedHorizontally>
+                <PrivateIndicator inWorkbench />
+              </BoxSpan>
+            )}
+            <RawButton onClick={toggleDisplay}>
+              <BoxSpan padded>
+                {isOpen ? (
+                  <StyledIcon
+                    styledHeight={"8px"}
+                    light
+                    glyph={Icon.GLYPHS.opened}
+                  />
+                ) : (
+                  <StyledIcon
+                    styledHeight={"8px"}
+                    light
+                    glyph={Icon.GLYPHS.closed}
+                  />
                 )}
-              </DraggableBox>
-            </Box>
+              </BoxSpan>
+            </RawButton>
           </Box>
-          {CatalogMemberMixin.isMixedInto(item) ? (
-            <Box centered paddedHorizontally>
-              {item.isPrivate && (
-                <BoxSpan paddedHorizontally>
-                  <PrivateIndicator inWorkbench />
-                </BoxSpan>
-              )}
-              <RawButton onClick={() => this.toggleDisplay()}>
-                <BoxSpan padded>
-                  {this.isOpen ? (
-                    <StyledIcon
-                      styledHeight={"8px"}
-                      light
-                      glyph={Icon.GLYPHS.opened}
-                    />
-                  ) : (
-                    <StyledIcon
-                      styledHeight={"8px"}
-                      light
-                      glyph={Icon.GLYPHS.closed}
-                    />
-                  )}
-                </BoxSpan>
-              </RawButton>
+        ) : null}
+      </Box>
+      {isOpen && (
+        <Box
+          column
+          gap={3}
+          paddedRatio={3}
+          css={{
+            borderTop: `1px solid ${theme.grey}`,
+            color: theme.greyLighter
+          }}
+        >
+          <WorkbenchItemControls item={item} viewState={viewState} />
+          {isLoading ? (
+            <Box paddedVertically>
+              <Loader light />
             </Box>
           ) : null}
         </Box>
-        {this.isOpen && (
-          <Box
-            column
-            gap={3}
-            paddedRatio={3}
-            css={{
-              borderTop: `1px solid ${this.props.theme.grey}`,
-              color: this.props.theme.greyLighter
-            }}
-          >
-            <WorkbenchItemControls
-              item={this.props.item}
-              viewState={this.props.viewState}
-            />
-            {isLoading ? (
-              <Box paddedVertically>
-                <Loader light />
-              </Box>
-            ) : null}
-          </Box>
-        )}
-      </StyledLi>
-    );
-  }
-}
+      )}
+    </StyledLi>
+  );
+});
+
+WorkbenchItemRaw.displayName = "WorkbenchItem";
 
 const DraggableBox = styled(Box)`
   cursor: move;
 `;
 
-const StyledLi = styled(Li)`
+const StyledLi = styled(Li)<{ theme: DefaultTheme }>`
```

Comment:
Is this extra typing required? I think it should be part of the default props type.

Reply:
VSCode was screaming at me for some reason, and I forgot to check after completing the changes. Thanks for noticing